src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -1094,15 +1097,29 @@ func (c controller) sendCompleteInstallation(ctx context.Context, isSuccess bool
 
 // logClusterOperatorsStatus logging cluster operators status
 func (c controller) logClusterOperatorsStatus() {
+	_, _ = c.operatorReport(true)
+}
+
+func (c controller) operatorReport(withLog bool) ([]models.OperatorMonitorReport, error) {
 	operators, err := c.kc.ListClusterOperators()
 	if err != nil {
 		c.log.WithError(err).Warning("Failed to list cluster operators")
-		return
+		return nil, err
 	}
 
+	report := make([]models.OperatorMonitorReport, 0)
 	for _, operator := range operators.Items {
-		c.log.Infof("Operator %s, statuses: %v", operator.Name, operator.Status.Conditions)
+		if withLog {
+			c.log.Infof("Operator %s, statuses: %v", operator.Name, operator.Status.Conditions)
+		}
+		status, _, reason := utils.MonitoredOperatorStatus(operator.Status.Conditions)
```

Comment:
Why do we drop the message? It contains a lot of detailed information, usually

Reply:
The log will be issued from the logger thread. In the postInstallConfig function we just need to collect the raw data to send to the service. So the log is printed just as before.

Reply:
Your response seems to imply you think I was talking about the `if withLog` block, but my comment is for the line immediately below that block, line 1115

Reply:
I did not drop anything really. The previous code just printed out the log in line 1113 and the line was kept as is. In addition to that I am assembling the report and the message was not used because it is very verbose. The requirements, as I got them from Avishay, were to keep the information as short as possible. In the BE I further omit extra info and prints only the operator names (so actually neither the reason nor the message is actually used).

Reply:
>I did not drop anything really. The previous code just printed out the log in line 1113 and the line was kept as is
I am not talking about the log line, I'm talking about the second return value of `MonitoredOperatorStatus`, that has the value `condition.Message`.

>In addition to that I am assembling the report and the message was not used because it is very verbose.

It's just 2-20 lines of text, what's wrong with it being verbose? It's very informative, why are we aiming to keep it short? Shouldn't keeping it short be a concern of the UI? It would be nice to have those messages / reasons saved in the backend and maybe even displayed to the user in the UI in the future

But if you've explicitly decided against including them then that's fine, you can unhold

Reply:
The point of the ticket was to send an event which is quite limited in what it can present in a readable fashion. After discussion it was decided to reduce the data presented to the user even further and print only the names of the failed operators. The field "reason" and "message" are quite interchangeable (you do not need them both). If you think that we should send the message instead I can certainly change the code here. In any case, it will not be presented in the UI in the current implementation - but may serve future endeavors.

Reply:
>The field "reason" and "message" are quite interchangeable

Interchangeable in what sense? They're very different

The "reason" is a very very short camel case "ID" of the error, it's almost meaningless for the user

The message is a few very informative lines of text that explain the issue in detail, and often include complete error messages which often contain clues to help the cluster owner understand what they can do to fix the problem

Reply:
Done.